brianlong/www.validators.app
# 184711419 rack attack whitelist

#### What's this PR do?
- Creates whitelist for selected users 

#### How should this be manually tested?
- Set user token in credentials. Send API requests using this token. You should not be blocked after exceeding the limit.

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/184711419)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
